scripts/hotrod-integration-test.sh

```diff
@@ -21,13 +46,15 @@ make create-baseimg
 
 # Loop through each platform (separated by commas)
 for platform in $(echo "$platforms" | tr ',' ' '); do
+  # Extract the operating system from the platform string
+  os=${platform%%/*}  #remove everything after the last slash
   # Extract the architecture from the platform string
   arch=${platform##*/}  # Remove everything before the last slash
-  make "build-all-in-one" GOOS=linux GOARCH="${arch}"
+  make "build-all-in-one" GOOS=${os} GOARCH="${arch}"
 done
 
 # Build image locally (-l) for integration test
-bash scripts/build-upload-a-docker-image.sh -l -c example-hotrod -d examples/hotrod -p "${platforms}"
+bash scripts/build-upload-a-docker-image.sh ${LOCAL_FLAG} -c ${BINARY} -d examples/hotrod -p "${platforms}"
```

Comment:
```suggestion
bash scripts/build-upload-a-docker-image.sh -l -c ${BINARY} -d examples/hotrod -p "${platforms}"
```
please re-read my previous comments

Reply:
My Bad, let me fix this